### PR TITLE
libical: 3.0.8 -> 3.0.9 (fix build on staging)

### PR DESCRIPTION
**Pull request by Mic92**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/libical/default.nix [hunk 1]
```diff
@@ -75,12 +75,6 @@ stdenv.mkDerivation rec {
     # Will appear in 3.1.0
     # https://github.com/libical/libical/issues/350
     ./respect-env-tzdir.patch
-    # Export src-generator binary for use while cross-compiling
-    # https://github.com/libical/libical/pull/439
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libical/default.nix:80]: Please remove fetchpatch from inputs.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/libraries/libical/default.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/development/libraries/libical/default.nix:80]: done.

**Comment by Mic92** [comment]
@GrahamcOfBorg eval

**Comment by Mic92** [comment]
ofborg you are drunk! If I compare the lines in the stacktrace they don't line up with the code from this repo!

**Pull request merged**